CAREERS.md

```diff
@@ -0,0 +1,15 @@
+🧳 Careers at daily.dev
+
+Are you looking for a new opportunity?
+We are a young and dynamic team with opportunities for amazing people like you to grow with our product.
```

Comment:
Agree with Ido, "We are a high-energy and dynamic team, looking for amazing people like you to grow alongside our product" as a suggestion?